yarn.lock

```diff
@@ -4625,9 +4625,9 @@ strip-json-comments@~2.0.1:
   version "2.0.1"
   resolved "https://registry.yarnpkg.com/strip-json-comments/-/strip-json-comments-2.0.1.tgz#3c531942e908c2697c0ec344858c286c7ca0a60a"
 
-stylis@3.0.15:
-  version "3.0.15"
-  resolved "https://registry.yarnpkg.com/stylis/-/stylis-3.0.15.tgz#e0f9bf403048511043b479d39422c835141e0fbb"
+stylis@^3.0.17:
```

Comment:
Sorry, that's the limit of what I can do on my phone 😉 I'll take care of this later when I'm in front​ if the PC.